ampoule/main.py

```diff
@@ -111,8 +117,15 @@ def main(reactor, ampChildPath):
     from twisted.application import reactors
     reactors.installReactor(reactor)
 
-    from twisted.python import log
-    log.startLogging(sys.stderr)
+    from twisted import logger
+    observer = logger.textFileLogObserver(sys.stderr)
+    logLevelPredicate = logger.LogLevelFilterPredicate(
+        defaultLogLevel=logger.LogLevel.info
+    )
+    filteringObserver = logger.FilteringLogObserver(
```

Comment:
Thanks for changing the bootstrapping as well!  One interesting question might be how to configure the filters on this predicate without having to put in a whole custom `BOOTSTRAP`.  For a future change, perhaps.

Reply:
That's a good thought.  I kind of hate that all this code lives in a big-old wad of text.  I can vaguely imagine some hand-wavey thing where we take command-line arguments, or an ampoule.cfg file, and generate the bootstrap code from a template.  Definitely bigger than the scope of this work, but it would have several benefits, including configuration of log levels and log formatters.

Reply:
> I kind of hate that all this code lives in a big-old wad of text

same

> an ampoule.cfg file, and generate the bootstrap code from a template

or you know, parse a datastructure passed in `argv` ?